packages/website/ts/components/relayer_index/relayer_grid_tile.tsx

```diff
@@ -125,11 +128,16 @@ class ImgWithFallback extends React.Component<ImgWithFallbackProps, ImgWithFallb
         };
     }
     public render(): React.ReactNode {
-        if (this.state.imageLoadFailed || _.isUndefined(this.props.src)) {
```

Comment:
I think this logic is useful and re-usable so probably should be in its own file. We can even call it `Image` and just have the fallback thing be part of the API.